ddl/ddl.go

```diff
@@ -1206,6 +1206,11 @@ func (d *ddl) CreateIndex(ctx context.Context, ti ast.Ident, unique bool, indexN
 		return errors.Trace(err)
 	}
 
+	// Deal with anonymous index.
+	if len(indexName.L) == 0 {
+		indexName = idxColNames[0].Column.Name
```

Comment:
If the is name conflict, MySQL automatically appends `_2` at the end;
```
create table t (c1 int, c2 int, index (c1,c2), index (c1));
```
The index name is `c1` for `(c1, c2)`, `c1_2` for `(c1)`